README.md

```diff
@@ -1,24 +1,101 @@
 # Validations in Controller Actions
 
-## Objectives
+Now that we know Rails automatically performs validations defined on models,
+let's use this information to easily display validation errors to the user.
```

Comment:
why do we want to be able to show validation errors to users? ie, what's an example use case for this?